File: AppGenerateFiles/dbo/Model/Catalogo_Sucursales.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Sucursales : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Sucursal { get; set; }
       public string? Nombre { get; set; }
       public string? Descripcion { get; set; }
       public string? Direccion { get; set; }
       public int? id_municipio { get; set; }
       [OneToMany(TableName = "Catalogo_Cuentas", KeyColumn = "Id_Sucursal", ForeignKeyColumn = "id_sucursal")]
       public List<Catalogo_Cuentas>? Catalogo_Cuentas_id_sucursal { get; set; }
       [OneToMany(TableName = "Catalogo_Cuentas", KeyColumn = "Id_Sucursal", ForeignKeyColumn = "id_sucursal")]
       public List<Catalogo_Cuentas>? Catalogo_Cuentas_id_sucursal { get; set; }
       [OneToOne(TableName = "Datos_Configuracion", KeyColumn = "Id_Sucursal", ForeignKeyColumn = "Id_Sucursal")]
       public Datos_Configuracion? Datos_Configuracion { get; set; }
       [OneToMany(TableName = "Security_Users", KeyColumn = "Id_Sucursal", ForeignKeyColumn = "Id_Sucursal")]
       public List<Security_Users>? Security_Users { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Sucursales.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Cuentas }  from './Catalogo_Cuentas.js'
import { Catalogo_Cuentas }  from './Catalogo_Cuentas.js'
import { Datos_Configuracion }  from './Datos_Configuracion.js'
import { Security_Users }  from './Security_Users.js'
class Catalogo_Sucursales extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Sucursal;
   /**@type {String}*/ Nombre;
   /**@type {String}*/ Descripcion;
   /**@type {String}*/ Direccion;
   /**@type {Number}*/ id_municipio;
   /**@type {Array<Catalogo_Cuentas>} OneToMany*/ Catalogo_Cuentas;
   /**@type {Array<Catalogo_Cuentas>} OneToMany*/ Catalogo_Cuentas;
   /**@type {Array<Datos_Configuracion>} OneToMany*/ Datos_Configuracion;
   /**@type {Array<Security_Users>} OneToMany*/ Security_Users;
}
export { Catalogo_Sucursales }
